engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeProjectContextTest.scala

```diff
@@ -12,6 +13,7 @@ class RuntimeProjectContextTest extends AnyWordSpec with Matchers {
     "report an exception if ran in context of a project " +
     "which cannot be loaded" in {
       val thrown = intercept[PolyglotException] {
+        LoggerSetup.get().setup()
```

Comment:
The necessity to call this `LoggerSetup.get().setup()` is annoying as also [noted elsewhere](https://github.com/enso-org/enso/pull/7803/files#r1326850557). The rule for these [injectable singletons](http://wiki.apidesign.org/wiki/Injectable_Singleton) is that they should be _configured automatically_ otherwise they become [pathological liars](http://wiki.apidesign.org/wiki/Singleton#Singletons_are_Pathological_Liars.5B3.5D).

Looks like it is not the case and that's wrong. We should find a way to convince whatever logging framework we use to use whatever configuration `LoggerSetup.get().setup()` without any call from tests. May we [provide [our own provider](https://www.slf4j.org/manual.html#swapping) to make sure a call to `LoggerSetup.get().setup()` is made?

Reply:
Provider was added, thanks for the suggestion!